pmm-app/tests/QAN/pages/qanPage.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
     showSelectedDisabled: '//button[@data-qa="qan-filters-show-selected" and @disabled ]',
     table: '//table//tr[2]',
     tableRow: 'div.tr',
-    noQueries: "//h1[contains(text(), 'No queries available for this combination of filters')]",
+    noQueries: '//h1[contains(text(), \'No queries available for this combination of filters\')]',
```

Comment:
Let's make sure we avoid the escaped quotes `\'` - replace them with double quotes `"`. It should work.
```suggestion
    noQueries: '//h1[contains(text(), "No queries available for this combination of filters")]',
```